source/images/simulator_state.svg

```diff
@@ -0,0 +1,37 @@
6 466,255.3545 466,261.5 C466,267.1174 453.9165,270.699 436.3184,272.2447 " fill="none" id="simulate-simulate" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="431.214,272.6267,440.4875,275.9436,436.2,272.2534,439.8903,267.9659,431.214,272.6267" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="77" x="472" y="266.5684">not terminal</text><!--link simulate to episode_terminal--><path d="M334.4207,294.2319 C335.6869,309.5014 334.5356,327.2857 326,341 C318.3977,353.2148 307.1164,362.8599 294.7327,370.428 " fill="none" id="simulate-episode_terminal" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="290.1251,373.1232,299.9133,372.0317,294.4409,370.5986,295.874,365.1263,290.1251,373.1232" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="67" x="333" y="337.5684">is terminal</text><!--link simulate to episode_terminal--><path d="M243.1058,294.0331 C231.4938,302.0843 221.1305,311.9589 214,324 C206.6478,336.4154 207.3047,352.1888 210.2627,365.8054 " fill="none" id="simulate-episode_terminal" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="211.4461,370.7142,213.2253,361.0274,210.2742,365.8535,205.4482,362.9024,211.4461,370.7142" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="102" x="215" y="337.5684">no more actions</text><!--link episode_terminal to episode_start--><path d="M153.1618,370.8596 C139.5406,363.0994 126.5108,353.2681 117,341 C107.8625,329.2135 103.0252,313.7062 100.4975,299.5194 " fill="none" id="episode_terminal-episode_start" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="99.667,294.3618,97.1486,303.8832,100.4619,299.2982,105.0469,302.6115,99.667,294.3618" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="81" x="118" y="337.5684">next episode</text><!--link episode_terminal to finished--><path d="M221,421.0911 C221,441.7035 221,470.9647 221,492.8484 " fill="none" id="episode_terminal-finished" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="221,497.961,225,488.961,221,492.961,217,488.961,221,497.961" style="stroke: #A80036; stroke-width: 1.0;"/><text fill="#000000" font-family="sans-serif" font-size="13" lengthAdjust="spacingAndGlyphs" textLength="112" x="222" y="464.5684">no more episodes</text><!--link finished to *end--><path d="M221,548.2229 C221,565.8366 221,588.7774 221,603.6928 " fill="none" id="finished-*end" style="stroke: #A80036; stroke-width: 1.0;"/><polygon fill="#A80036" points="221,608.8002,225,599.8002,221,603.8002,217,599.8002,221,608.8002" style="stroke: #A80036; stroke-width: 1.0;"/><!--
```

Comment:
I am confused by this file. It looks like PlantUML which would have the extension of `.uml` and be a UTF-8 encoded text file, but it also has declarations for SVG. Would this even render?

Reply:
it's svg output. the original plantuml is embedded in the svg as a comment.

Reply:
👍